auto_save.py

```diff
@@ -56,7 +56,7 @@ def callback():
       '''
       Must use this callback for ST2 compatibility
       '''
-      if view.is_dirty() and not view.is_loading():
+      if view.is_dirty() and not view.is_loading() and not view.is_auto_complete_visible():
         if not backup: # Save file
```

Comment:
Is there a command in the new api like view.is_quick_panel_visible()?

This is also an issue if the quick panel is up it will automatically save and close it.